jetty-start/pom.xml

```diff
@@ -69,7 +72,6 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-util</artifactId>
       <version>${project.version}</version>
```

Comment:
Is this dependency still needed ?

Reply:
Yes. Without it the shared plugin will not shade any dependencies.  So jetty-start is now normally dependent on jetty-util, but we just choose to use the jar with shaded dependencies

Reply:
Can it not this dependency be put inside the plugin dependencies ?
Because `jetty-start` does not depend on `jetty-util` so this appear to me to be a dependency that is only needed to the shade plugin, not to the `jetty-start` module.

Reply:
Jetty-start uses two classes from jetty-util, so it is dependent on it. It will not compile or run with out this dependency

Reply:
`jetty-start` in 9.4.8 did not depend on `jetty-util`.

The dependency was introduced when the shaded plugin was introduced.

So either we make `jetty-start` depend explicitly on `jetty-util`, and we don't shade; or we shade but don't explicitly make `jetty-start` depend on `jetty-util`.

Or, I don't understand what this change is about :)

Reply:
reactor build order is based on project.dependencies, plugin dependencies are not taking into account.   
Without the build.dependencies.dependency its easy for the reactor to decide to build jetty-start before jetty-util.

Reply:
You cannot have jetty-start just use jetty-util as-is.  The classloader it builds will introduce jetty-util again.  You now have duplicate classes.  What if you want to release a point fix for ${jetty.home}/lib/jetty-util-<version>.jar ?

Reply:
@sbordet I think you don't understand :)  there were to duplicated classes in jetty-start. I have deleted them and now normally depend on jetty-util for those. However we don't want to use the normal artifact as it's jetty-util dependency will introduce the problems that @joakime describes.  But we don't use the normal artifact, we b use the shaded artifact with no other dependencies.  So there are no problems like joakim suggests. 

@joakime there are no duplicate jars because we don't use the normal artifact. Jetty-util just takes the shaded jar and everything works as before. There is no problem using the component classes because they are small and shaded.  I'm just not seeing any problems with this approach. It is no different to hire we use the shaded jetty client jar in some places to avoid duplicates.